File: Android_Programiranje3_Ontologija-master/app/src/main/java/ffos/p3/ontologija/Ontologija.java
package ffos.p3.ontologija;

import java.io.Serializable;

public class Ontologija implements Serializable {

    public Ontologija() {
        this.sifra = sifra;
        this.nazivPodcasta = nazivPodcasta;
        this.voditelj = voditelj;
        this.brojPregleda = brojPregleda;
        this.podcastTrajanje = podcastTrajanje;
        this.uriPodcasta = uriPodcasta;
    }

    private int sifra;
    private String nazivPodcasta;
    private String voditelj;
    private Integer brojPregleda;
    private String podcastTrajanje;
    private String uriPodcasta;

    public int getSifra() {
        return sifra;
    }

    public void setSifra(int sifra) {
        this.sifra = sifra;
    }

    public String getNazivPodcasta() {
        return nazivPodcasta;
    }

    public void setNazivPodcasta(String nazivPodcasta) {
        this.nazivPodcasta = nazivPodcasta;
    }

    public String getVoditelj() {
        return voditelj;
    }

    public void setVoditelj(String voditelj) {
        this.voditelj = voditelj;
    }

    public Integer getBrojPregleda() {
        return brojPregleda;
    }

    public void setBrojPregleda(Integer brojPregleda) {
        this.brojPregleda = brojPregleda;
    }

    public String getPodcastTrajanje() {
        return podcastTrajanje;
    }

    public void setPodcastTrajanje(String podcastTrajanje) {
        this.podcastTrajanje = podcastTrajanje;
    }

    public String getUriPodcasta() {
        return uriPodcasta;
    }

    public void setUriPodcasta(String uriPodcasta) {
        this.uriPodcasta = uriPodcasta;
    }
}

File: OntologijaP3-API/src/Kolaric/Ontologija.php
<?php

namespace Kolaric;

/**
 * @Entity @Table(name="ontologija")
 **/


class Ontologija 

{
    /** @id @Column(type="integer") @GeneratedValue **/
    protected $sifra;

    /**
    * @Column(type="string")
    */
    private $nazivPodcasta;

   /**
    * @Column(type="string")
    */
    private $voditelj;

    /**
    * @Column(type="integer")
    */
    private $brojPregleda;

    /**
    * @Column(type="string")
    */
    private $podcastTrajanje;


    /**
    * @Column(type="string")
    */

    private $uriPodcasta;



    

    public function getSifra(){
      return $this->sifra;
    }
  
    public function setSifra($sifra){
      $this->sifra = $sifra;
    }
  
    public function getNazivPodcasta(){
      return $this->nazivPodcasta;
    }
  
    public function setNazivPodcasta($nazivPodcasta){
      $this->nazivPodcasta = $nazivPodcasta;
    }
  
    public function getVoditelj(){
      return $this->voditelj;
    }
  
    public function setVoditelj($voditelj){
      $this->voditelj = $voditelj;
    }
  
    public function getBrojPregleda(){
      return $this->brojPregleda;
    }
  
    public function setBrojPregleda($brojPregleda){
      $this->brojPregleda = $brojPregleda;
    }
  
    public function getPodcastTrajanje(){
      return $this->podcastTrajanje;
    }
  
    public function setPodcastTrajanje($podcastTrajanje){
      $this->podcastTrajanje = $podcastTrajanje;
    }
  
    public function getUriPodcasta(){
      return $this->uriPodcasta;
    }
  
    public function setUriPodcasta($uriPodcasta){
      $this->uriPodcasta = $uriPodcasta;
    }

    

    


    public function setPodaci($podaci)
    {
      foreach($podaci as $kljuc => $vrijednost){
        $this->{$kljuc} = $vrijednost;
      }
    }
    
  }

?>